tests/dragonfly/snapshot_test.py

```diff
@@ -583,38 +583,25 @@ async def test_big_value_serialization_memory_limit(df_factory, query):
     instance.start()
     client = instance.client()
 
-    ten_mb = 10_000_000
+    one_gb = 1_000_000_000
+    elements = 1000
+    one_mb = 1_000_000
 
-    def ten_mb_random_string():
-        return "".join(random.choices(string.ascii_letters, k=ten_mb))
-
-    one_gb = 1_000_000_000  # 1GB
-
-    upper_limit = one_gb * 1.1  # 1GB + 100MB
-
-    i = 0
-
-    while instance.rss < one_gb:
-        if query == "HSET":
-            i = i + 1
-            await client.execute_command(f"HSET foo_key foo_{i} {ten_mb_random_string()}")
-        elif query == "SADD":
-            await client.execute_command(f"SADD foo_key {ten_mb_random_string()}")
-        elif query == "ZSET":
-            await client.execute_command(f"ZADD foo_key {i} {ten_mb_random_string()}")
-        elif query == "LIST":
-            await client.execute_command(f"LPUSH foo_key {ten_mb_random_string()}")
-
-    async def check_memory_usage(instance):
-        while True:
-            assert instance.rss < upper_limit
-            await asyncio.sleep(0.01)
-
-    checker = asyncio.create_task(check_memory_usage(instance))
+    await client.execute_command(
+        f"debug populate 1 prefix {one_mb} TYPE {query} RAND ELEMENTS {elements}"
+    )
 
+    info = await client.info("ALL")
+    # rss double's because of DEBUG POPULATE
```

Comment:
what do you mean, why does it doubles because of DEBUG POPULATE?

Reply:
see my other comment